### PR TITLE
list subscriptions

### PR DESCRIPTION
**Pull request by RavindraDesai042**

### CODE REVIEW

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl [hunk 1]
```diff
@@ -28,6 +27,66 @@
...
+                            </a>
+                        <#else>
+                            <a href="#"
+                               data-target="#activateEmployeeConfirmModal"
+                               class="btn btn-outline-primary" title="Enable" data-toggle="modal"
+                               data-party-id="${orgPartyId}" data-party-name="${subscription.productId!}"
+                               data-org-party-id="${orgPartyId!}">
+                                <i class="fa fa-unlock" aria-hidden="true"></i>
+                            </a>
+                        </#if>
```
Comment (RavindraDesai042) [inline apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/pages/customers/tabs/customer_subscriptions.ftl]: ignore this, I'll refactor next

File: apache-ofbiz-16.11.05/hot-deploy/admin/servicedef/services.xml [hunk 2]
```diff
@@ -62,5 +62,14 @@ under the License.
         <attribute name="orderId" type="String" mode="OUT" optional="false"/>
     </service>
 
+    <service name="listSubscriptions" engine="java"
```
Comment (harishbnaritha) [inline apache-ofbiz-16.11.05/hot-deploy/admin/servicedef/services.xml]: service name should not have "list" or "show" kind of words. thats something UI does.
Services should be operations like "get", "fetch", "load", etc.
For this, lets call it getSubscriptions

File: apache-ofbiz-16.11.05/hot-deploy/admin/src/main/java/com/autopatt/admin/services/SubscriptionServices.java [hunk 3]
```diff
@@ -107,4 +109,38 @@
         return sendResp;
     }
 
+    public static Map<String, Object> listSubscriptions(DispatchContext ctx, Map<String, ? extends Object> context) {
```
Comment (harishbnaritha) [inline apache-ofbiz-16.11.05/hot-deploy/admin/src/main/java/com/autopatt/admin/services/SubscriptionServices.java]: same naming change please

File: apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/WEB-INF/actions/customers/customer_subscriptions.groovy [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+String orgPartyId = parameters.orgPartyId
+
+context.orgPartyId = orgPartyId;
+
+// Get list of subscriptions for this customer
+List subscriptions =new ArrayList();
+resp = dispatcher.runSync("listSubscriptions", UtilMisc.<String, Object>toMap("orgPartyId", orgPartyId,
+        "userLogin", userLogin));
+
+if (resp != null && resp.get("responseMessage") != null) {
```
Comment (harishbnaritha) [inline apache-ofbiz-16.11.05/hot-deploy/admin/webapp/admin/WEB-INF/actions/customers/customer_subscriptions.groovy]: No need to check responseMessage.
We should use ServiceUtil.isSuccess(resp) method, it handles everything

**Pull request merged**